ably/rest/auth.py

```diff
@@ -154,7 +157,7 @@ def request_token(self, key_name=None, key_secret=None, query_time=None,
         return TokenDetails.from_dict(response_json)
 
     def create_token_request(self, key_name=None, key_secret=None,
-                             query_time=False, token_params=None):
+                             query_time=None, token_params=None):
```

Comment:
Is it idiomatic to have all of these arguments as separate named arguments instead of a single `authOptions` argument (which is merged from the given arguments and the instance-wide defaults)? It seems a bit fragile that each time we add a new member to `AuthOptions` then we have to change the declaration of every function that takes these options as arguments.